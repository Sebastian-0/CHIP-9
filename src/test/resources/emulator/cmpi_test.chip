ldi A, 5

# Equal
clrflag
cmpi 5
sout FLAGS

# Less
clrflag
cmpi 6
sout FLAGS

# Greater
clrflag
cmpi 4
sout FLAGS

hcf