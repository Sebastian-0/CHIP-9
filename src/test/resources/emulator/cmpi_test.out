160
80
32